doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
+#########################
+
+Many packages in Refkit come with a default set of configuration files.
+This document gives guidance for configuring a Refkit image with custom
+package configuration files.
+
+Using .bbappend files to override package configuration
+=======================================================
+
+If you have only a limited set of packages which you need to configure,
+you can create ``.bbappend`` files which append the recipes for each
+package which you want to modify. A usual way to change the
+configuration is to change the install phase to replace or remove the
+default configuration files.
+
+As explained in `yocto-compatible.rst <yocto-compatible.rst>`_, the
```

Comment:
I must second mythi's comment here: this "Yocto Compatible" requirement only applies to .bbappends contained in refkit itself and thus to Refkit core developers. Users of refkit can do whatever they want in their custom distro (configuration) layer.